src/parser.pegjs

```diff
@@ -21,7 +21,7 @@
 //
 // [1] http://www.ecma-international.org/publications/standards/Ecma-262.htm
 
-{
+{{
```

Comment:
you must have bootstrapped this?

Reply:
Yes, I generated the parser with a per-parse initializer first then transformed it into a global initializer and re-generated. It served as a first test ;)